Source/RxSwift/ObservableType+Weak.swift

```diff
@@ -29,6 +29,28 @@ extension ObservableType {
             method?(obj)()
         }
     }
+    
+    /**
+     Performs a flatMap providing a safe unretained reference to `obj` as an argument of the flatMap closure
```

Comment:
Let's use the description of original flatMap, something like 

> Projects each element of an observable sequence to an observable sequence and merges the resulting observable sequences into one observable sequence, while providing a safe & unretained reference to the provided object.

Reply:
done